dev/com.ibm.ws.webcontainer.security.app/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -86,3 +89,12 @@ includePathInWASReqURL.desc=Setting the Path parameter can allow the client/brow
 
 trackLoggedOutSSOCookies=Track logged out LTPA single signon tokens
 trackLoggedOutSSOCookies.desc=Specifies whether to track LTPA single signon tokens that are logged out on a server so that it can not be reused on the same server.
+
+overrideHttpAuthenticationMechanism=Name of the container provided HttpAuthenticationMechanism to override the application configured HttpAuthenticationMechanism. The HttpAutheMechanism is defined by the Java EE 8 Security specification.
+overrideHttpAuthenticationMechanism.desc=Specifies HttpAuthenticationMechanism name for overriding an application defined HttpAuthenticationMechanism. The acceptable value is BASIC or FORM. When FORM is used, loginFormURL and loginErrorURL need to be specified.
+
+loginFormContextRoot=Context root for the login form.
+loginFormContextRoot.desc=Specifies the context root of a form login page which is specified by loginFormURL. If this value is not set, the first path element of the loginFormURL is used as a context root. This value is applicable for JavaEE security only.
+
+httpBasicAuthMechanismRealmName=Realm name fot the basic authentication to be used when overrideHttpAuthenticationMechanism is enabled and set to to BASIC.
```

Comment:
Suggested change:
The realm name for basic authentication when the overrideHttpAuthenticationMechanism container is enabled and set to BASIC.